snuba/migrations/groups.py

```diff
@@ -17,6 +17,7 @@ class MigrationGroup(Enum):
     SESSIONS = "sessions"
     QUERYLOG = "querylog"
     SPANS_EXPERIMENTAL = "spans_experimental"
+    PROFILES = "profiles"
```

Comment:
Please add this migration to the skipped migration group for now https://github.com/getsentry/snuba/blob/master/snuba/settings/__init__.py#L157. Since we are still experimenting we do not want the migration to be run for Self hosted or On-Prem installations